sdk/core/azure-core/inc/azure/core/uuid.hpp

```diff
@@ -74,12 +74,11 @@ namespace Azure { namespace Core {
     static Uuid CreateUuid()
     {
       std::random_device rd;
-      std::mt19937 gen(rd());
 
       uint8_t uuid[UuidSize] = {};
 
       for (int i = 0; i < UuidSize; i += 4)
-        *reinterpret_cast<uint32_t*>(uuid + i) = gen();
+        *reinterpret_cast<uint32_t*>(uuid + i) = rd();
```

Comment:
Thanks for the feedback.  I added this.  I switched the random_device to avoid reseeding the generator on each function call.  The docs lead me to believe this is the right pattern, can you confirm.

Reply:
Yes, this is correct with the caveat that `random_device` is sadly not guaranteed to be portably random (http://eel.is/c++draft/rand.device#2). We could assert that `entropy()` returns something greater than `0.0` in the tests for a bit of added paranoia.